docs/index.rst

```diff
@@ -85,6 +85,24 @@ your ~/.bash_profile::
 
 Note that a functional NetCDF install is required.
 
+Windows specific installation instructions
+------------------------------------------
+
+To enable Parcels to run on a Windows machine, some extra steps have to be taken to install the necessary prerequisite files. Windows must have the following software before it is able to run Parcels:
+* A working distribution of gcc must be installed. With working is meant that gcc an be accessed from the commandline. We suggest Windows users to install the `MinGW Package <www.mingw.org>`_. Do note that to access the gcc compiler which MinGW provides, the user must most likely add the directory path to the 'MinGW\bin' folder where gcc resides.
+* A version of python with a working copy of Numpy and SciPy. We recommend you to install a scientific Python distribution, like `anaconda <https://anaconda.org/anaconda/python>`_, which provides the easier way to install Numpy and SciPy. However, it is also possible to use a version of Python distributed by the Python organisation (make sure you grab the version 2 branch of Python). After installing Python you can install the packages manually by downloading the appropriate versions of these packages from this repository of `pre-built Windows installers <https://www.lfd.uci.edu/~gohlke/pythonlibs/>`_. Use the command: python -m pip install 'package' to install these modules.
+
```

Comment:
I would only mention the miniconda option. I think that we can assume that users who want to run Parcels on Windows will not want the added hassle of having to download their own scipy/numpy etc?